chingu-voyages/v50-tier2-team-14
# added images/logo folder in public to store logo files

# Description
Added logo svg and png files to the project folder

## Main changes explained:
-Created new images/logo folder in public
-Added 2 files with logo (2 versions) and 2 files with marker (svg and png).

## Screenshot:

![Screenshot 2024-07-22 at 11 54 49 AM](https://github.com/user-attachments/assets/1fb214f9-a3a6-4d84-972b-7f44507b3fdf)

## How to test:
1. check into current branch
2. check project folder structure
3. try to use the logo file in your component
4. verify that you can access and use the logo and marker files
5. if you think it's better to store these files in a different folder, let me know in comments
